libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -194,6 +194,26 @@ uint16_t SRV_Channel::pwm_from_range(float scaled_value) const
     return servo_min + uint16_t( (scaled_value * (float)(servo_max - servo_min)) / (float)high_out );
 }
 
+// convert a pwm to a 0..1 value
+float SRV_Channel::range_from_pwm() const
```

Comment:
It might be nice to call this "scaled from pwm" and have it work for angle type too. Don't necessarily need to do it now. It would just be nice to check that this is configured as a range type `type_setup && !type_angle`.